tests/benchmarks/fixture-bootstrap.php

```
@@ -0,0 +1,445 @@
+
+			$packages = WC()->shipping()->calculate_shipping( WC()->cart->get_shipping_packages() );
+			$rates    = [];
+			foreach ( $packages as $package ) {
+				$rates = array_merge( $rates, (array) ( $package['rates'] ?? [] ) );
+			}
+
+			if ( empty( $rates ) ) {
+				throw new RuntimeException( 'Benchmark fixture setup failed: no WooCommerce shipping rates are available for the benchmark cart.' );
+			}
+
+			$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'assert_preflight'. (undefined)

(StaticAccess)